structuredefinitions/UKCore-MedicationDispense.xml

```diff
@@ -4,9 +4,9 @@
   <url value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-MedicationDispense" />
   <version value="2.4.0" />
   <name value="UKCoreMedicationDispense" />
-  <title value="UK Core MedicationDispense" />
+  <title value="UK Core Medication Dispense" />
```

Comment:
Don't think this should be changed because 'MedicationDispense' is named as such in base standard.